waiter/config-full.edn

```diff
@@ -777,14 +777,14 @@
 
  ; Waiter maintains a list of recently "killed" and "erroneous" (e.g. an error occurred while streaming the response)
  ; instances. Such instances are guaranteed not to be served up as an available instance until sufficient time has
- ; elapsed since their last use. This is referred to as blacklisting an instance:
- :blacklist-config {
-                    ;; Erroneous instances are blacklisted using an exponential delay based on the number of successive
-                    ;; failures and blacklist-backoff-base-time-ms (milliseconds):
-                    :blacklist-backoff-base-time-ms 10000
-
-                    ;; Killed instances are blacklisted for max-blacklist-time-ms (milliseconds):
-                    :max-blacklist-time-ms 300000}
+ ; elapsed since their last use. This is referred to as ejecting an instance:
+ :eject-config {
```

Comment:
`eject` is the verb, `ejection` is the process by which something becomes `ejected`, but after discussion, use of `eject` instead of `ejection` is probably good enough and avoids the annoying choice of whether to use `ejection` or `eject` in a bunch of places.